src/config.rs

```diff
@@ -2364,7 +2383,8 @@ impl ConfigHandler {
                         "error" => ?e,
                     );
                     version = configpb::Version::default();
-                    incoming = local_config;
+                    incoming =
+                        get_last_config(&local_config.storage.data_dir).unwrap_or(local_config);
```

Comment:
So last_cfg has higher priority than the local config? Make sense.

Reply:
`local config` (i.e default config or config file from terminal) are intended use when tikv startup first time, after that `PD` will manger the tikv config and provide restart config, and `last_cfg` are intended use when the restart config `PD` provided is invalid. cc @overvenus